cypress/lib/mocks.js

```diff
@@ -1,10 +1,18 @@
 export const mockApi = () => {
 	// Mock share count
-	cy.intercept('GET', '/sharecount/**', { fixture: 'shareCount.json' });
+	cy.intercept('GET', '/sharecount/**', {
+		fixture: 'shareCount.json',
+	}).as('getShareCount');
 	// Mock most-read
-	cy.intercept('GET', '/most-read/**', { fixture: 'mostRead.json' });
+	cy.intercept('GET', '/most-read/**', {
+		fixture: 'mostRead.json',
+	}).as('getMostRead');
 	// Mock most-read
-	cy.intercept('GET', '**/most-read-geo**', { fixture: 'mostReadGeo.json' });
+	cy.intercept('GET', '**/most-read-geo**', {
+		fixture: 'mostReadGeo.json',
+	}).as('getMostReadGeo');
 	// Mock most-read
-	cy.intercept('GET', '/embed/card/**', { fixture: 'richLink.json' });
+	cy.intercept('GET', '/embed/card/**', {
+		fixture: 'richLink.json',
+	}).as('getRichLink');
```

Comment:
I suspect these aliases are not needed. I don't see them being used in the PR so I think they can be removed.

Reply:
They're needed for the tests to work. If you remove them it fails and that's why the test was previously failing. It couldn't find the aliases and that caused the tests to fail. I re-ran this multiple times with multiple configurations to check